scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
The script is meant to authenticate and provide the necessary tokens for the system test driver so that test can use the data to login to the cluster. 
I've created a new directory as none of the test utils is executable but would move it into `test_utils` if you think that's the appropriate location.